wallets/arweave-wallet.md

```diff
@@ -84,4 +84,10 @@ To remove a set of wallet keys from the wallet app, click the <img src="../.gitb
 
 From this screen you'll be able to re-download the key file for a particular wallet key pair or click the "Delete" button to remove it from the wallet app entirely. As long as you have your passphrase or key file you'll be able to add it back whenever you like so keep those safe and secure.
 
+## **Using Arweave.app as a cold wallet**
```

Comment:
this needs to stay in FAQs. it's the most popular question, and has even got it's own page.